druid-shell/src/backend/windows/window.rs

```diff
@@ -185,18 +185,18 @@ impl Eq for WindowHandle {}
 unsafe impl HasRawWindowHandle for WindowHandle {
     fn raw_window_handle(&self) -> RawWindowHandle {
         if let Some(hwnd) = self.get_hwnd() {
-            let handle = WindowsHandle {
-                hwnd: hwnd as *mut core::ffi::c_void,
-                hinstance: unsafe {
-                    winapi::um::libloaderapi::GetModuleHandleW(0 as winapi::um::winnt::LPCWSTR)
-                        as *mut core::ffi::c_void
-                },
-                ..WindowsHandle::empty()
+            // Using empty + set fields instead Win32Handle{...} to shut up a unexplainable compiler error
```

Comment:
I wouldn't say it's "unexplainable." `Win32Handle` is marked as `#[non_exhaustive]`, which is explained [here](https://doc.rust-lang.org/reference/attributes/type_system.html)

Reply:
Ah, that explains it. Thanks.